### DISCUSSION TITLE
kbin Magazine, and name suggestion

### DISCUSSION THREAD

**Discussion by artillect** [aclist/kbin-megamod#15] [Archived]
Since this is a project for kbin, I think it's only natural that we make a magazine for it once it's ready for people to try out. However, I think we could change the name to something a bit catchier. As much as I like the current name, I think something like kbin Enhancement Suite conveys what it is a bit more effectively, and it's instantly clear what we're trying to do for people familiar with RES. We could create /m/enhancement if we go with that, but if not, I think /m/egamod could be fun (although linking that would look funky).

**Comment by minnieoo** [comment]
i have to agree. i do like the name megamod, it sounds cool. however for the sake of clarity and familiarity for the average kbin user (which is a reddit migrant), kbin Enhancement Suite would be a lot more attractive to them.

**Comment by aclist** [comment]
While imitation is the sincerity form of flattery, I tend to feel that using the name of other projects can be a bit too derivative, but I guess the name is generic enough, it's not anything proprietary like a brand name.

So KES? KEM? kbin enhancement menu? kbin magic menu? kbin addon menu?
Any recursive GNU-style acronym we can make here? 

I'm basically indifferent on the matter, will defer to what you want. I chose "mod" as jargon because "userscript" is basically meaningless to a non-technical person.

As for a magazine, I made a few subreddits in the past to manage support issues with software I released, and it became logistically overwhelming to maintain, because you also have to deal with culling spam, bots, unrelated posts, questions with zero troubleshooting info ("This is broken, what do I do?"), etc, so it's a bit of a Pandora's box. I tend to want to lead people to the issues page because we can use issue templates that instruct them to list specific information such as browser, addon version, log output, etc, and it filters the questions at the top level and massages them into something more manageable. I'm not against it per se, but be warned that it might cause fragmentation of support channels and can be time-consuming (depending on how many people actually use the plugin).

With all that said, I guess if you want to decide on a name and make the magazine, then add us as mods?

**Reply by minnieoo** [reply]
I do think the name "enhancement suite" is generic enough to not be considered derivative (in a bad way) while also being familiar enough to attract attention. People will think "oh okay, this is like RES but for Kbin, it will just enhance my experience." which leaves no room for confusion (though, seeing how people responded to the fediverse, there will probably be confusion no matter what you do lol). Speaking of non-technical people, they might see the word mod and think of something like game mods, or assume it's a menu for moderators (I actually assumed this when seeing the name the first time), who knows lol. 

But, if you'd like to be different while still drawing from the familiarity of RES, there is tons of potential changes that could be made to differentiate itself while still being inspired and clear. These are rough, but for example:

- Kbin Elite Suite
- Kbin Advanced Suite
- Kbin Power Pack
- Kbin Deluxe
- Kbin Pro Suite
- Kbin Boost
- Kbin Performance Suite
- Kbin Plus
- Kbin Pro Enhancer
- Kbin Ultra
- Kbin Master Suite
- Kbin Mega Suite
- Kbin Super (supercharge?)

In the end, I don't know how important it would be to go from Kbin Megamod to an RES-like name, but it has become clear that some people really need to be handheld through things to get it without writing it off completely 😅. Either way would probably turn out fine in the end.

**Reply by aclist** [reply]
@artillect I forgot to add that the `/m/egamod` thing was cute, nice one.

@minnieoo That's a fair point, hadn't thought about that with the word mod.

All of the above sound fine, let's put it to a vote or whatever due process is necessary here. KES seems fine. What about just Kbin Enhancer?

Some other stupid ideas:

- AFK (Addons For Kbin)
- Kbin Community Fixer
- KUTE (Kbin Userscript and Theme Enhancements)

Sorry, I'm terrible at these. I'm partial to something succinct, but Kbin Plus and Kbin Ultra sound like commercial products.

Basically, what would someone say when they think, "OK, I just need to pull up the <NAME> menu"?

**Reply by minnieoo** [reply]
> Basically, what would someone say when they think, "OK, I just need to pull up the menu"?

In that case, KES and Addons for Kbin sound fine and clear (though the abbreviation AFK might confuse others when there's conversation about it since it has another meaning). KUTE is too long.

**Reply by artillect** [reply]
Yeah, I agree that KES is generic enough that we're not stepping on their toes. We're also following in their footsteps, so I feel like it's somewhat appropriate.

> As for a magazine, I made a few subreddits in the past to manage support issues with software I released, and it became logistically overwhelming to maintain, because you also have to deal with culling spam, bots, unrelated posts, questions with zero troubleshooting info ("This is broken, what do I do?"), etc, so it's a bit of a Pandora's box.

I've never moderated a sub for a project of mine, but I do have a lot of experience doing moderation in general, so I could handle it. I just went ahead and made [/m/enhancement](https://kbin.social/m/enhancement), since it seems like we are going with the name KES (although shouldn't it technically be kES since kbin is stylized with a lowercase k?).

> I forgot to add that the /m/egamod thing was cute, nice one.

Haha, ty. I always loved seeing that on reddit, it's not the most clever thing in the world but it's fun.

**Reply by aclist** [reply]
Thanks for doing that, sounds good. Now time to find somewhere in the menu to cram a link back to the magazine. (I feel like full screen modal is the inevitable outcome we are careening towards)

**Reply by artillect** [reply]
Might be a good idea to add a home page, for stuff like links to the magazine, this repo, etc. 

I think it'd be fun to have a logo too, I sorta like the [bird logo](https://kbin.social/m/kbinMeta/t/80273/Here-s-the-nice-Kbird-with-changes-that-people-suggested-is#comments) that FixedFun came up with. I could make something similar to that, although it isn't really 100% related to the theme of kbin (although does it really have a consistent theme lol?) Maybe the kbin logo but with some random "upgrades" attached to it? We could just go with /kES in a nice font as the logotype for now, here's a couple I threw together real quick:
![KES logo lowercase](https://github.com/aclist/kbin-megamod/assets/8906201/5b395cbe-e82b-4009-b53b-8a1968ffa9ee)
![KES logo](https://github.com/aclist/kbin-megamod/assets/8906201/63703681-78b7-4aaa-93ab-a51a314c6fb7)

**Reply by aclist** [reply]
I just had a dumb idea. What about KAP, Kbin Addons Panel, and it's the bird wearing a backwards baseball cap? Is that dumb? Lol.

Otherwise, lowercase k seems good. Maybe take a look at Pragmata Pro, it's a monospace font but has a nice elongated look.

**Reply by artillect** [reply]
KAP is kinda cute with that logo idea! Personally I prefer KES over that since I feel like KAP implies that it's just a config panel

**Reply by aclist** [reply]
I just wanted a bird wearing a hat :(

Could probably still use the icon in the header

**Reply by artillect** [reply]
After seeing these absolutely incredible 3D-rendered [magazine logos](https://kbin.social/m/kbinMeta/t/136524/I-designed-and-rendered-a-couple-of-icons-for-the#comments) by @fearout@kbin.social, I decided to take a shot at making one of my own for this

It's nowhere near as good as theirs, but I'm starting to dig it. 
![the kbin Enhancement Suite Logo, a 3D render of the kbin logo with a wrench and screwdriver in the folder](https://github.com/aclist/kbin-megamod/assets/8906201/816529be-e8dd-44e7-9e01-ddc9cc6f144d)

**Reply by aclist** [reply]
Interesting! That wheel looks kinda funny though lol

We actually have a relevant ticket here: https://github.com/aclist/kbin-megamod/issues/46

**Reply by artillect** [reply]
Yeah I was having trouble getting the wheel to look just right, I think there's an easier way to do it that looks nicer. Either way, I think it'd be fun to have a 3D logo, and if we could get fearout to do it that'd be amazing.

**Reply by aclist** [reply]
Is there a transparent version? We could add it to the documentation file I am writing

**Reply by artillect** [reply]
I can put that together, are there any other things you think I should add to it? I think it needs to have more buttons/switches lol

**Reply by aclist** [reply]
Hmm, needs more knobs of some sort on the side in lieu of the wheel, something that looks like a +/- volume knob? And something on the front face? I'm kind of at a loss for that spot. Maybe some exposed machinery? Some kind of CSS sheet? This is a hard one, I feel like the messaging is a bit mixed here. Is this an icon about working with tools or about upgrading your kBin? (rhetorical question)

**Reply by artillect** [reply]
> Hmm, needs more knobs of some sort on the side in lieu of the wheel, something that looks like a +/- volume knob? 

I like the idea of the knob on the side, the wheel is a bit hard to get looking nice with what little space there is to work with.

> And something on the front face? I'm kind of at a loss for that spot. Maybe some exposed machinery? Some kind of CSS sheet?

I was thinking maybe a screen of some sort, but that might be a bit too busy. Maybe some more knobs and some sliders? 

> I feel like the messaging is a bit mixed here. Is this an icon about working with tools or about upgrading your kBin? (rhetorical question)

Good point, the tools were there from when I started working on this earlier, before I had a more solid idea of what it should look like. I agree, I think they'd probably be better off for more of a "toolbox" sort of extension. The emphasis should definitely be on upgrading/configuring kbin, I'll lean more into that.

**Reply by aclist** [reply]
I'm just drawing a blank on this right now, waiting for inspiration to come. I don't think it's too busy to fill in the face; that's what it's there for. I guess let's just keep workshopping it until we have some eureka moment. Maybe something less literal and more metaphorical? E.g., "hotrodding" your kBin to mod it. Maybe you could add car spoilers and other effects to the folder to make it look like it was augmented.
